packages/mux-player/src/template.ts

```diff
@@ -78,7 +78,7 @@ export const content = (props: MuxTemplateProps) => html`
       player-software-name="${props.playerSoftwareName ?? false}"
       player-software-version="${props.playerSoftwareVersion ?? false}"
       env-key="${props.envKey ?? false}"
-      stream-type="${props.streamType ?? false}"
+      stream-type="${props.streamType ? (isLiveOrDVR(props) ? 'live' : 'on-demand') : false}"
```

Comment:
@cjpillsbury fixed the issue causing the tests to fail here for now

Reply:
praise: Good catch here.